package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "vui-grid-system",
+  "version": "0.0.1",
+  "description": "Mixins for laying out elements on a grid.",
+  "scripts": {
+    "build:css:gutters": "node-sass --output-style expanded ./sample/gutters.css.scss ./sample/gutters.css",
+    "build:css:simple": "node-sass --output-style expanded ./sample/simple.css.scss ./sample/simple.css",
+    "build:css:isolate": "node-sass --output-style expanded ./sample/isolate.css.scss ./sample/isolate.css",
+    "postinstall": "bower install",
+    "test": "exit 0"
```

Comment:
I might suggest running the build as part of the `test` script, just so Travis actually does something and catches errors.